docs/api/sql/Function.md

```diff
@@ -2940,9 +2940,17 @@ POINT (-70.01 44.37)
 
 ## ST_Union
 
-Introduction: Return the union of geometry A and B
+Introduction:
+
+Variant 1: Return the union of geometry A and B.
```

Comment:
1. Can you also update the ST_H3toGeom and ST_S2toGeom doc to explain how this ST_Union works in conjunction with these functions?

2. What happens to polygons that could not be unioned (polygons that share no boundaries)?

Reply:
1. Sure.
2. It returns a MultiPolygon with each set of boundaries. See here: https://github.com/apache/sedona/blob/ec6dfff79974a2846ecff5dcdf6a95417a7cb7c9/spark/common/src/test/scala/org/apache/sedona/sql/functionTestScala.scala#L700-L705

Reply:
@furqaankhan Cool. Can you also explain this in the doc?